### ISSUE TITLE
Mirroing side cameras but not rear camera results in odd continuity

### ISSUE THREAD

**Issue by Mardak** [ehendrix23/tesla_dashcam#102]
The default fullscreen layout with mirror results in some oddness if someone walks past the car from behind as the person appears at the left edge of the left camera then moves to the right edge of the left camera then jumps to the right edge of the rear camera moving to the left edge of the rear camera then jumps to the left edge of the right camera and then moves to the right edge of the right camera:

```
Example circle moving from the driver side to the passenger side:
t1 [o..][...][...]
t2 [.o.][...][...]
t3 [..o][...][...]
t4 [...][..o][...]
t5 [...][.o.][...]
t6 [...][o..][...]
t7 [...][...][o..]
t8 [...][...][.o.]
t9 [...][...][..o]
```

Not sure if the rear camera should also be mirrored as if looking through the rear view mirror.

Or perhaps if the mirrored side cameras are placed at the top next to the forward cameras and perhaps that will make it feel less like there should be continuity from left to rear to right cameras.

**Comment by ehendrix23** [comment]
Changed in release 0.1.16

**Comment by Mardak** [comment]
Ah fixed in a9cf2406e2ab43ec33ae4ceb30913251d70d3a9a thanks.
